Dockerfile

```
@@ -94,6 +94,8 @@ COPY --from=assets-precompile /app /app
 COPY --from=assets-precompile /usr/local/bundle/ /usr/local/bundle/
 COPY --from=middleman /public/ /app/public/
 
+RUN echo "cd /app && /usr/local/bundle/bin/bundle exec rails c" > /root/.ash_history
```

also to remove the painful /usr/local/bin
```suggestion
ENV BUNDLE_PATH "/usr/local/bundle"
ENV PATH "$PATH:/usr/local/bin"
```

---

I think this is a better fix. echo'ing the command in the history, is probably helpful but does not address the root of the problem which is that `bundle` in not in the `PATH`